audio-player.js

```diff
@@ -230,14 +230,14 @@ function formatText(text) {
 
 function updateDisplay() {
```

Comment:
@jsonpoindexter do think it would be a good idea to pass the `selectedMood` and `selectedEnvironment` as arguments to this function, and ensure those are defined?

Reply:
I think it's ok since its already referencing other global variables and there is an undefined/null check in the function for `selectedMood`. Do we need one for `selectedEnviorment`?

Reply:
thank you @jsonpoindexter I just pushed a check. let me know what you think!

Reply:
Looks great thanks!

Reply:
thank you @jsonpoindexter ! is there anything else that might need adjustment before your approval, or could I go ahead and merge this into develop so that I can use jsdelivr from there?

Reply:
since I got your thumbs up on the part of the code that changed, I am going to go a head and merge, and we can revert if needed :) @jsonpoindexter